spring-rabbit/src/main/java/org/springframework/amqp/rabbit/config/ListenerContainerFactoryBean.java

```diff
@@ -389,6 +392,10 @@ public void setReceiveTimeout(long receiveTimeout) {
 		this.receiveTimeout = receiveTimeout;
 	}
 
+	public void setBatchReceiveTimeout(long batchReceiveTimeout) {
```

Comment:
Add Javadoc here, please.
To be short: every public API must be with Javadocs.
I know, we have many missed, but let's cover at least whatever new we add so far!